config/brakeman.ignore

```diff
@@ -107,7 +153,7 @@
       "check_name": "Redirect",
       "message": "Possible unprotected redirect",
       "file": "app/controllers/reports_controller.rb",
-      "line": 21,
+      "line": 25,
```

Comment:
Is it worth keeping these ordered by line number?

Reply:
This is autogenerated by brakeman - I suppose we could, but it'll mean constantly fighting against the tool to do it...

Reply:
Given it's not really for human consumption I'd prefer to leave it. Thoughts?